src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -28,7 +29,11 @@ public void clearResults() {
 
     @Test
     public void getStatisticAboutApple() {
-        workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        try {
+            workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        } catch (IOException e) {
+            System.out.println("Can't write data to file");
+        }
```

Comment:
The try-catch block for `IOException` is not necessary in test methods. Letting the exception propagate will cause the test to fail and provide more detailed information about the failure.